src/controllers/modals/ActionModalController.php

```diff
@@ -266,20 +266,83 @@ class="fb-cta cta--yellow js-close-modal js-confirm-save">
           <h4>
             {tr('account_')}<span class="highlighted">{tr('Settings')}</span>
           </h4>;
+        $oauth_header = '';
+        if (Configuration::genFacebookOAuthSettingsExists() === true) {
+          $linked = \HH\Asio\join(
```

Comment:
Not in this PR, but in general, there should be one call to `\HH\Asio\join()` per HTTP request; that'll need some refactoring of the model classes.

Reply:
+1 on this!